performance_regression_cdc_test.py

```diff
@@ -0,0 +1,269 @@
+        self.wait_no_compactions_running()
+        self.run_fstrim_on_all_db_nodes()
+        self.truncate_base_table(node1)
+        node1.run_cqlsh(f"ALTER TABLE {self.keyspace}.{self.table} WITH cdc = {{'enabled': true}}")
+
+        _ = self._workload_cdc(write_cmd, 2,  # pylint: disable=unused-variable
+                               test_name="test_mixed",
+                               sub_type="cdc_enabled",
+                               cdc_read_cmd=read_cdc_log_cmd)
+
+        self.wait_no_compactions_running()
+        self.run_fstrim_on_all_db_nodes()
+        self.truncate_base_table(node1, cdclog_table=True)
+        node1.run_cqlsh(f"ALTER TABLE {self.keyspace}.{self.table} WITH cdc = {{'enabled': true, 'preimage': true}}")
+
+        _ = self._workload_cdc(write_cmd, 2,  # pylint: disable=unused-variable
+                               test_name="test_mixed_with_cdc_preimage",
+                               sub_type="cdc_preimage_enabled",
+                               cdc_read_cmd=read_cdc_log_cmd)
+        self.wait_no_compactions_running()
+
+        self.check_regression_with_baseline(base_line="cdc_disabled")
+
+    def _workload_cdc(self, stress_cmd, stress_num, test_name, sub_type=None, keyspace_num=1, prefix='', debug_message='',  # pylint: disable=too-many-arguments,too-many-locals
+                      save_stats=True, cdc_read_cmd=None):
+        read_statistics = None
+        future = None
+        if debug_message:
+            self.log.debug(debug_message)
+
+        if save_stats:
+            self.create_test_stats(sub_type=sub_type,
+                                   doc_id_with_timestamp=True,
+                                   append_sub_test_to_name=True)
+
+        stress_queue = self.run_stress_thread(stress_cmd=stress_cmd, stress_num=stress_num, keyspace_num=keyspace_num,
+                                              prefix=prefix, stats_aggregate_cmds=False)
+        if cdc_read_cmd:
+            loader_node: BaseNode = self.loaders.nodes[0]
+            loader_node.remoter.run('go get -u github.com/piodul/cdc-stressor')
+            db_node: BaseNode = self.db_cluster.nodes[0]
+
+            def read_cdc_log(loader: BaseNode):
+                result = loader.remoter.run(
+                    f'{cdc_read_cmd} -nodes {db_node.ip_address}')
+                return result
+            with ThreadPoolExecutor(max_workers=1) as executor:
+                future = executor.submit(read_cdc_log, loader_node)
+
+        results = self.get_stress_results(queue=stress_queue, store_results=True)
+        if future:
+            read_statistics = future.result()
+            self.log.info(read_statistics.stdout)
+
+        if save_stats:
+            self.update_test_details(scylla_conf=True)
+            stat_results = PP.pformat(self._stats["results"])
+            self.log.debug(f'Results {test_name}: \n{stat_results}')
+            self.display_results(results, test_name=test_name)
```

Comment:
why do we need this function ? is anyone uses those .xml files ?

Reply:
Didn't see, but it used on main performance tests, so leave it as is

Reply:
please remove it we don't use it